spec/models/glider_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Glider, type: :model do
+  fixtures :all
+
+  describe '#name' do
+    context 'glider name' do # how to improve this?
+      subject(:glider) do
+        Glider.new(glider_type: 'Wave 2', size: 'S', manufacturer: 'Skyjam')
+      end
+
+      it 'returns "Wave 2 | Skyjam"' do # should I use '' and "" the other way around?
```

Comment:
It's up to you. Really, the text for the `it` method call is just for documentation of what the test is supposed to test.